src/applications/personalization/dashboard/components/benefit-payments-and-debts/BenefitPaymentsAndDebt.jsx

```diff
@@ -83,27 +89,33 @@ const BenefitPaymentsAndDebt = ({
               icon="dollar-sign"
               newTab
               text="Manage your direct deposit"
+              /* eslint-disable react/jsx-no-bind */

```

Comment:
onClick param does require a callback-function, so I'm disabling the eslint-warning.